api/routes/userRoutes.py

```diff
@@ -28,12 +28,18 @@ def get_user_by_id(id: str):
 @userRoutes.route('/user', methods=['POST'])
 def create_user():
     data = request.get_json()
-
-    new_user = User(
-        username=data.username,
-        email=data.email,
-        password=data.password
+    # print(data)
+
+    new_user = Users(
+        # username=data.username,
+        # email=data.email,
+        # password=data.password
+        username=data["username"],
+        email=data["email"],
+        password=data["password"]
```

Comment:
You can do research on your own between object.property and object["property"], both is fine, but synchronization would be better for the project